### PR TITLE
Push UI updates

### PR DESCRIPTION
**Pull request by DavidRollins**
### For internal users - Please add a Jira DX PR ticket to the subject!

---

### Preview Link
https://deploy-preview-5488--tyk-docs.netlify.app/docs/nightly/getting-started/quick-start/

### Description
1. Describe your changes in detail. Why is this change required? What problem does it solve?
     2. Please explain the change to the reviewer. Pay special attention to changes that are hard to spot, 
       for example:
       2.1. Updating menu.yaml file to new IA structure, should not lose any content from master.
       2.2. Updating html, css, and js objects to fix mobile platform and UI for new IA.
     3. @mentions of the person to review the proposed changes. They need to be able to know the topic well in order to approve it.

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

### CODE REVIEW

File: tyk-docs/static/js/docs-navigation.js [hunk 1]
```diff
@@ -103,10 +103,10 @@ var doNav = function() {
 		let extension = $('[data-filetype]').data('filetype') || 'md';
 		
     if ( $('.active').hasClass('st-open')){
-			window.open("https://github.com/TykTechnologies/tyk-docs/tree/master/tyk-docs/content/" + githubCustomLink + '/' + docName + '.md', "_blank");
+			window.open("https://github.com/TykTechnologies/tyk-docs/tree/master/tyk-docs" + githubCustomLink + '/' + docName + '.md', "_blank");
 		} else if ( $('.active').hasClass('st-file') && $('.active').closest('.st-open').length === 0) {
 			if(currentPageGH == '/docs'){
-				window.open("https://github.com/TykTechnologies/tyk-docs/tree/master/tyk-docs/content/documentation.md", "_blank");
+				window.open("https://github.com/TykTechnologies/tyk-docs/tree/master/tyk-docsdocumentation.md", "_blank");
```
Comment (sharadregoti) [inline tyk-docs/static/js/docs-navigation.js:109]:
```suggestion
				window.open("https://github.com/TykTechnologies/tyk-docs/tree/master/tyk-docs/documentation.md", "_blank");
```

**Review by sharadregoti — CHANGES_REQUESTED** [review]

**Comment by yurisasuke** [comment]
@DavidRollins . One ui issue.The left navigation is too squeezed (the width seems too small)

image

File: tyk-docs/static/css/docs.css [hunk 2]
```diff
@@ -928,11 +928,11 @@ iframe {
   display: none !important;
 }
 
-.toc-content {
+toc-content {
   height: 100%;
 }
 
-.toc-label {
+toc-label {
```
Comment (buger) [inline tyk-docs/static/css/docs.css:935]: Was the goal to just disable those styles, by removing the dot?

Comment (buger) [inline tyk-docs/static/css/docs.css:935]: Was the goal to just disable those styles, by removing the dot?

File: tyk-docs/static/css/docs.css [hunk 3]
```diff
@@ -21,10 +21,10 @@
   background-color: white;
   z-index: 11;
   width: 100%;
-  padding-left: 50px;
-  padding-right: 50px;
-  padding-top: 62px;
-  padding-bottom: 85px;
+  padding-left: 20px;
```
Comment (letzya) [inline tyk-docs/static/css/docs.css:24]: i think we've done that

**Comment by letzya** [comment]
@DavidRollins closing it so it's not in our queue, thanks

**Pull request closed without merging**